values.yaml

```diff
@@ -690,4 +690,5 @@ custom:
         ## Optional. Job annotations
         # annotations: {}
         ## Optional. Job pod annotations
-        # podAnnotations: {}
\ No newline at end of file
+        # podAnnotations: {}
+  #projects: []
```

Comment:
add docs on what fields are available in projects and their valid values. Refer to the above custom paths for doc reference

Reply:
Updated values.yaml